src/scalar.rs

```diff
@@ -256,18 +257,10 @@ impl Scalar {
     /// - `Some(s)`, where `s` is the `Scalar` corresponding to `bytes`,
     ///   if `bytes` is a canonical byte representation;
     /// - `None` if `bytes` is not a canonical byte representation.
-    pub fn from_canonical_bytes(bytes: [u8; 32]) -> Option<Scalar> {
-        // Check that the high bit is not set
-        if (bytes[31] >> 7) != 0u8 {
-            return None;
-        }
+    pub fn from_canonical_bytes(bytes: [u8; 32]) -> CtOption<Scalar> {
```

Comment:
This change should be documented in the changelog.

Reply:
Added in c51ba24